app/controllers/admins/dashboards_controller.rb

```diff
@@ -9,8 +9,8 @@ def design
 
   def index
     @service_requests = admin_requests.page(params[:page])
-    @open_service_requests = admin_requests.where(status_id: Status.where(name: 'Abierto').last).count#select(&:open?).count
-    @closed_service_requests =  admin_requests.where(status_id: Status.where(name: 'Cerrado').last).count#.select(&:closed?).count
+    @open_service_requests = admin_requests.where(status_id: 1).count#select(&:open?).count
+    @closed_service_requests =  admin_requests.where(status_id: 2).count#.select(&:closed?).count
```

Comment:
Line is too long. [97/80]<br>Unnecessary spacing detected.<br>Missing space after #.<br>Operator `=` should be surrounded by a single space.<br>Put a space before an end-of-line comment.